### DISCUSSION TITLE
Move logo to $panel array config

### DISCUSSION THREAD

**Discussion by aurawindsurfing** [filamentphp/filament#7195] [Ideas]
I think It would make sense to move logo to $panel array just like with favicon.

https://beta.filamentphp.com/docs/3.x/panels/themes#changing-the-logo

```
return $panel
  ->favicon(asset('images/favicon.png'))
  ->logo(asset('images/panel_logo.png'))
```

**Comment by danharrin** [comment]
Disagree, it would make resizing, especially responsive resizing, difficult. Favicons don't have those concerns.

**Comment by aurawindsurfing** [comment]
Understood.

Is there a way of having different logos for different panels at the moment?

**Comment by aurawindsurfing** [comment]
Probably I can just detect the panel id and display correct logo this way.

**Reply by danharrin** [reply]
yeah in the view
